crates/webidl/src/update_cargo_toml.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::{Context, Result};
+use std::{fs, path::PathBuf};
+
+pub fn update_cargo_toml_features(
+    cargo_toml_path: &PathBuf,
+    generated_features: &str,
+) -> Result<()> {
+    let mut cargo_toml_text = fs::read_to_string(cargo_toml_path).context("reading Cargo.toml")?;
+    let comment = "# This list is auto-generated by the wasm-bindgen-webidl program";
+    let features_head = "[features]";
+    let features_table = format!(
+        r"{comment}
+{features_head}
+{generated_features}
+"
+    );
+
+    match cargo_toml_text.find(features_head) {
+        Some(features_start) => {
+            let comment_start = cargo_toml_text[..features_start]
+                .rfind(comment)
+                .unwrap_or(0);
+
+            let end = cargo_toml_text[features_start..]
+                .find("\n[")
```

Comment:
I'm assuming this is for finding a new TOML table header (not sure about the terminology here)?
Maybe add a comment here.

Reply:
Good idea, I put some comments for understandability.